terraform/rancher/cluster/logging.tf

```diff
@@ -108,13 +126,12 @@ data:
 
     # Send the logs to the standard output
     <match **>
-      @type elasticsearch_dynamic
+      @type elasticsearch
       include_tag_key true
       host "#{ENV['ELASTICSEARCH_HOST']}"
       port "#{ENV['ELASTICSEARCH_PORT']}"
       scheme http
       logstash_format true
-      logstash_prefix logstash-${record["kubernetes"]["namespace_name"]}
```

Comment:
This is Kseniia changes about filtering logs by namespace, so please bring this changes back

Reply:
Done. Reverted my commit.